File: LeetCode/java/1051_Height_Checker.java
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

class Solution {
    public int heightChecker(int[] heights) {
        int answer = 0;
        List<Integer> expected = new ArrayList<>();

        for (int n : heights) {
            expected.add(n);
        }

        Collections.sort(expected);

        for (int i = 0; i < heights.length; i++) {
            if (heights[i] != expected.get(i)) {
                answer++;
            }
        }

        return answer;
    }
}

File: LeetCode/cpp/1051_Height_Checker.cpp
class Solution {
public:
	int heightChecker(vector<int>& heights) {
		int answer = 0;
		vector<int> expected;

		for (int n : heights) {
			expected.push_back(n);
		}

		sort(expected.begin(), expected.end());

		for (int i = 0; i < heights.size(); i++) {
			if (heights[i] != expected[i]) {
				answer++;
			}
		}

		return answer;
	}
};